configuration/debian/tedge_agent/postinst

```diff
@@ -13,6 +13,6 @@ if command -v systemctl >/dev/null; then
 fi
 
 # Initialize the agent
-runuser -u tedge -- tedge_agent --init
+sudo -u tedge -- tedge_agent --init
```

Comment:
On debian, this will run as root. I checked on my system that `sudo` doesn't request for root password when run undo root. But I failed to find this in the documentation. Do you have any clue on that?

Reply:
not atm, I'll check that.

Reply:
After searching through the internet I have not found any solution regarding `sudo`, however I might have found solution regarding using `runuser` on yocto embedded system. I will share the results after the weekend.

Reply:
I think that `sudo` should work - I just want to be sure. I will check.